e2e/cypress/integration/account_settings/theme_color/custom_colors_sidebar_styles_spec.js

```diff
@@ -147,3 +147,102 @@ describe('AS14318 Theme Colors - Color Picker', () => {
         });
     });
 });
+
+describe('MM-14318 Theme Colors', () => {
+    before(() => {
+        //Login and navigating to settings
+        cy.apiLogin('user-1');
+        cy.toAccountSettingsModal(null, true);
+        cy.get('#displayButton').click();
+        cy.get('#themeTitle').click();
+        cy.get('#customThemes').click();
+    });
+
+    after(() => {
+        cy.defaultTheme();
+    });
+
+    it('Should be able to use color picker input and change Sidebar theme color', () => {
+        cy.get('#SidebarStyles').click();
+
+        // # Click the Sidebar BG setting
+        cy.get('.color-icon:visible').first().click();
```

Comment:
Can you slap an id or data-testid attribute on this element in the react code?

Reply:
Done 👍